docs/src/prototypes/IconViewer/index.tsx

```diff
@@ -94,7 +94,7 @@ class IconViewerExample extends React.Component<any, {}> {
                         .sort()
                         .map(name => (
                           <div key={`${name}-outline`} style={cellStyles}>
-                            <Icon name={name} variables={{ outline: true }} />
+                            <Icon name={name} outline />
                             <br />
                             <code>{name.replace(processedIconsNamePrefix, '')} outline</code>
```

Comment:
I don't  like that here we are kind a suggesting that the outline is part of the name :\ Not related to your changes, but would rather see something like `<Icon name=`${name.replace(processedIconsNamePrefix, '')}` outline />` or some object `{name: `${name.replace(processedIconsNamePrefix, '')}`, outline: true}`, because I was asked multiple times by people about what the name `like outline` for example for the Icon's doesn't work. It is not obvious that the outline is something else...

Reply:
![copy](https://user-images.githubusercontent.com/14183168/53645465-9a40f580-3c41-11e9-9598-f5fb95c949be.gif)

Added copy 😺